docs/usage/deep-linking.md

```diff
@@ -1,19 +1,20 @@
 # `deepLinking` parameter
 
-Swagger-UI allows you to deeply link into tags and operations within a spec. When Swagger-UI is provided a URL fragment at runtime, it will automatically expand and scroll to a specified tag or operation.
+Swagger-UI allows you to deeply link into tags and operations within a spec. When a deep link is clicked, it will automatically expand and scroll to a specified tag or operation.
 
 ## Usage
 
-👉🏼 Add `deepLinking: true` to your Swagger-UI configuration to enable this functionality.
+👉🏼 Add `deeplinking: true` to your Swagger-UI configuration to enable this functionality.
```

Comment:
Change needed: `deepLinking` is case sensitive, and should be preserved throughout this PR